swagger.yaml

```
@@ -10,22 +10,26 @@ basePath: "/v1"
 schemes:
   - http
 tags:
-  - name: "hello"
   - name: "private"
+  - name: "public"
 paths:
-  /hello:
+  /legacy:
```

I believe this route is missing a segment:

```suggestion
  /releases/legacy:
```